bears/general/LineCountBear.py

```diff
@@ -17,16 +18,25 @@ def _get_blank_line_count(self, file):
             list(filter(lambda x: re.match(r'^\s*$', x), file)))
         return num_blank_lines
 
-    def run(self, filename, file, max_lines_per_file: int,
+    def run(self, filename, file, min_lines_per_file: int,
+            max_lines_per_file: int=1000,
```

Comment:
Still, separate commit for this one

Reply:
@Makman2 What do you understand by new commit? :) My second commit in this PR or new issue for newcomers ? :)